web/css/style.css

```diff
@@ -49,10 +48,14 @@ h1{
 	transform: translateX(-50%);
 }
 
-.second-button{
+.top-button{
```

Comment:
not going to force you into this one since my reviews are getting length, but if this class will only ever be with action-button it may make sense to declare it as `.action-button.top-button`

Reply:
same for `bottom-button`